services/cd-service/pkg/service/overview.go

```diff
@@ -732,50 +733,113 @@ func (o *OverviewServiceServer) StreamDeploymentHistory(in *api.DeploymentHistor
 		}
 
 		err = stream.Send(&api.DeploymentHistoryResponse{
-			Deployment: "time,app,environment,deployed release version,previous release version\n",
+			Deployment: "time,app,environment,deployed release version,source repository commit hash,previous release version\n",
 			Progress:   uint32(100 / (count + 1)),
 		})
 		if err != nil {
 			return err
 		}
-
+		type ReleaseInfo struct {
+			appName        string
+			releaseVersion uint64
+		}
+		startDateReleases := in.StartDate.AsTime().AddDate(0, -1, 0)
+		var releases = make(map[ReleaseInfo]string)
+		//Get releases for which we found any relevant deployment. We want to extract the commit hash for that release
 		query := o.DBHandler.AdaptQuery(`
+			SELECT appName, metadata, releaseVersion FROM releases
```

Comment:
Not sure, but shouldn't we look at the `releases_history` table instead. For example if a release is deleted, do we want to count it in the compliance or not?